server/bin/pbench-audit-server.sh

```diff
@@ -715,10 +716,11 @@ function verify_archive {
     return $cnt
 }
 
-# Ensure we have an existing final report file.
-report=${LOGSDIR}/${PROG}/report.latest.txt
-report_prev=${LOGSDIR}/${PROG}/report.prev.txt
-mv ${report} ${report_prev} 2>/dev/null
+# Save the previous report.
+latest_report=${LOGSDIR}/${PROG}/report.latest.txt
+prev_report=${LOGSDIR}/${PROG}/report.prev.txt
+mv ${latest_report} ${prev_report} 2>/dev/null
```

Comment:
I'm not sure how this works unconditionally.  Ignoring the bootstrapping problem (where there isn't a pre-existing "latest report", because we've never run before or because the disk got wiped), after line 722 we no longer have a "latest report" and there won't be one for the next run to move unless we make it to line 778.